.nvmrc

```diff
@@ -0,0 +1 @@
+8.15
```

Comment:
Ha, was going to suggest adding this as I got my bearings in seriate repo. Good call.